core/app/models/spree/shipment.rb

```diff
@@ -36,6 +36,7 @@ def generate_number(options = {})
     scope :ready,   -> { with_state('ready') }
     scope :shipped, -> { with_state('shipped') }
     scope :trackable, -> { where("tracking IS NOT NULL AND tracking != ''") }
+    scope :editable, -> { where(state: ["pending", "ready"]) }
```

Comment:
@athal7 I agree but I think this is a bigger discussion. For now at least this scope follows Spree's current behaviour and provides a hook that you can override for finding editable shipments. We will follow this PR soon with another one that deals with this issue separately. That cool?